backends/qualcomm/quantizer/custom_annotation.py

```diff
@@ -118,3 +118,29 @@ def annotate_matmul_input1(node: Node, quantization_config: QuantizationConfig):
                 if "SDPA" in full_qualified_name:
                     annotate_matmul(node, quantization_config_16a8w)
                     annotate_matmul_input1(node.args[1], quantization_config_8a8w)
+
+
+def custom_annotate_matmul_16a8w(gm: torch.fx.GraphModule):
```

Comment:
I feel like we can add this annotation...what do you think?

Reply:
yes, it looks good to me.

Reply:
Create [this PR](https://github.com/pytorch/executorch/pull/5864) separately so we can merge them while continuing improving the quality